.drone.star

```diff
@@ -855,6 +855,94 @@ def buildRelease(ctx):
 		}
 	]
 
+def website(ctx):
```

Comment:
What about calling this "docs" instead of "website"?

Reply:
it is called `website` in all our repos. It publishes the content to `owncloud.github.io`